cognite/client/data_classes/workflows.py

```diff
@@ -1230,6 +1232,36 @@ def _load_trigger(cls, data: dict) -> Self:
         raise NotImplementedError()
 
 
+class WorkflowTriggerDataModelingQuery(CogniteObject):
+    """This class represents a data modeling trigger query."""
+
+    with_: dict[str, ResultSetExpression]
+    select: dict[str, Select]
+
+    def __init__(
+        self,
+        with_: dict[str, ResultSetExpression] = {},
+        select: dict[str, Select] = {},
```

Comment:
Mutable default arguments can not be used here

Reply:
removed